src/commands/lambda/functions/instrument.ts

```diff
@@ -135,16 +147,20 @@ export const calculateUpdateRequest = async (
   let needsUpdate = false
 
   // Update Handler
-  const expectedHandler = HANDLER_LOCATION[runtime]
-  if (config.Handler !== expectedHandler) {
-    needsUpdate = true
-    updateRequest.Handler = HANDLER_LOCATION[runtime]
+  if (runtime !== DOTNET_RUNTIME && runtime !== 'java11' && runtime !== 'java8.al2' && runtime !== 'provided.al2' && runtime !== 'ruby2.5' && runtime !== 'ruby2.7') {
```

Comment:
This seems like a rather long `if` statement. Is there a way to make this a bit shorter and easier to read?